src/Files.App/Helpers/ExternalResourcesHelper.cs

```diff
@@ -71,13 +72,13 @@ public async Task LoadOtherThemesAsync()
 
 		private async Task AddThemesAsync(StorageFolder folder)
 		{
-			foreach (var file in (await folder.GetFilesAsync()).Where(x => x.FileType == ".xaml"))
+			foreach (var file in (await folder.GetFilesAsync()).Where(x => string.Equals(x.FileType, ".xaml", StringComparison.InvariantCultureIgnoreCase)))
 			{
-				if (!Themes.Exists(t => t.AbsolutePath == file.Path))
+				if (!Themes.Any(t => t.AbsolutePath == file.Path))
 				{
 					Themes.Add(new AppTheme()
 					{
-						Name = file.Name.Replace(".xaml", "", StringComparison.Ordinal),
+						Name = file.Name[..^5],
```

Comment:
What does `^5` do? I don't think I've seen this before.

Reply:
It's trimming the last 5 entries from a list (here is trimming the last 5 char from a string).